### DISCUSSION TITLE
Compatability with Quantum Space Buddies (Multiplayer mod)

### DISCUSSION THREAD

**Discussion by JpTiger** [Raicuparta/nomai-vr#386] [General]
I haven't tested this, but it would be incredible to be able to play this game VR in multiplayer.

**Comment by JpTiger** [comment]
Just to get folks on the same page, potentially: https://github.com/misternebula/quantum-space-buddies/issues/246

**Comment by loco-choco** [comment]
I'm not that knowledgeable about the inner workings of either mods, but at their current state, a support from either is difficult, mostly because of the harmony patches that they have. _nebula is working and reworking stuff, so _maybe_ some kind of API can be made available, and then someone can try _connecting_ the mods. But still, they both change lots of stuff, and a conflict is almost inevitable.

**Comment by Raicuparta** [comment]
I can say that there will be no effort from me to make this compatible. Even when I first created the QSB project, I had no intention of making them compatible. Both mods require modifying code in many, many parts of the game. So conflicts are very likely to happen. Would just be too difficult to keep them working together. The mods are already hard enough as they are.

I'd say the best chance this has of seeing the day of light would be if someone creates a Virtual Reality Space Buddies that merges both mods into one. Probably only after QSB is more complete.

In any case, I'm not actively developing NomaiVR anymore, so I wouldn't do something this big anyway.

**Comment by misternebula** [comment]
> I can say that there will be no effort from me to make this compatible. Even when I first created the QSB project, I had no intention of making them compatible. Both mods require modifying code in many, many parts of the game. So conflicts are very likely to happen. Would just be too difficult to keep them working together. The mods are already hard enough as they are.

Agreed. And while continuing making QSB (Rai made up to version 0.1.2 I think? and it's on 0.7.0/0.8.0 currently), I certainly haven't been thinking about VR compatibility.

**Comment by Raicuparta** [comment]
@misternebula is here 😍 I'm a huge fan will you sign my repo

**Comment by misternebula** [comment]
🖋️ : _nebula
